### ISSUE TITLE
Add the sim. feature

### ISSUE THREAD

**Issue by s11141827** [musescore/MuseScore#19716]
### Your idea

So my idea was to add another text symbol called sim. so that way instead of repeating the dynamics, articulations, & any other music symbols, I just put it there & it plays it back as written

### Problem to be solved

Being able to add sim. while playing it back w/o having to repeat the symbols.

**Comment by Jojo-Schmitz** [comment]
Adding such a _sim._ text would be very easy, getting it to playback all the variuos things it might apply to, dynamics, artictulations etc. it the difficult, if not even impossible, part

**Comment by s11141827** [comment]
> Adding such a _sim._ text would be very easy, getting it to playback all the variuos things it might apply to, dynamics, artictulations etc. it the difficult, if not even impossible, part

Yeah it would take time to develop.

**Comment by Jojo-Schmitz** [comment]
It is not as simple as "look back and repeat the last dynamic, ornament or articulation" at least, as it could be several articulations, e.g. staccato, tennuto, staccato, tenuto

**Comment by MarcSabatella** [comment]
Indeed, and then some AI to figure out how to apply this.  Like if you correctly guess it was meant to apply to the articulations and the pattern was dot-dot-dash-dot over four eighths, how would that apply to measures with quarters, or rests.  Or if it’s pedal markings, how to analyze the harmonies to decide where is the “similar” place to lift.

It’s really not a solvable problem.

**Comment by s11141827** [comment]
> Indeed, and then some AI to figure out how to apply this. Like if you correctly guess it was meant to apply to the articulations and the pattern was dot-dot-dash-dot over four eighths, how would that apply to measures with quarters, or rests. Or if it’s pedal markings, how to analyze the harmonies to decide where is the “similar” place to lift.
> 
> It’s really not a solvable problem.

It would have to be made up of several versions like sim articulations, sim pedal, etc.

**Comment by yezhiyi9670** [comment]
I think a practical and reliable solution is to make the user select what to repeat. Some concerns could still remain (especially if articulations are taken into account), but they seem possible to solve.
